src/experimental/tools/mediaCapabilitiesProber/probers/mediaContentType/index.ts

```diff
@@ -60,6 +60,6 @@ export default function probeContentType(config: IMediaConfiguration): Promise<n
         const support = (window as any).MediaSource.isTypeSupported(val) ? 2 : 0;
         return Math.min(acc, support);
       }, 3);
-      return result;
+      return [result] as [number];
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=315&issues=AWXy2D-Y2IDoRcZ5Haqp&open=AWXy2D-Y2IDoRcZ5Haqp)

[ISSUE_KEY]: AWXy2D-Y2IDoRcZ5Haqp